src/components/Messages/Message/index.js

```diff
@@ -91,8 +97,9 @@ const renderContent = ({
 		),
 	text && (
 		<MessageBubble inverse={me} msgSequence={msgSequence} quoted={quoted} system={system}>
-			{showPostChatUrl && <Button onClick={() => onClickFeedback()} className = {createClassName(styles, 'surveyButton__content')}>Give Feedback</Button>}
+			{showPostChatUrl && <Button onClick={() => onClickFeedback()} className = {createClassName(styles, 'closedChatButton__content')}> { I18n.t('Give Feedback') } </Button>}
 			<MessageText style={showPostChatUrl && { width: '300px' }} text={text} system={system} />
+			{isChatClosed && store.state.config.settings.transcript && <Button onClick={() => onClickTranscript()} className = {createClassName(styles, 'closedChatButton__content')}> { I18n.t('Save Transcript') } </Button>}
 		</MessageBubble>
```

Comment:
@AlexanderKanakis Can you change the order of components here, please?
1) End chat text
2) Give Feedback button
2) Save transcript button